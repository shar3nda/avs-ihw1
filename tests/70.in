10
-1 7 2 6 5 10 10 7 2 1