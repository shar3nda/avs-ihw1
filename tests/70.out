-7 4 -6 4 8 10 -1 -2 1 -10